finality-aleph/src/justification.rs

```diff
@@ -45,6 +45,9 @@ impl AlephJustification {
     }
 }
 
+pub type JustificationsCadence = Duration;
+pub type ChainCadence = (SessionPeriod, MillisecsPerBlock, JustificationsCadence);
```

Comment:
Furthermore, it seems that the `JustificationsCadence` can be calculated from the other two fields, so maybe this should be a method?

Reply:
This tuple/struct came about only because of clippy warning you when a function has more than 7 arguments :shrug:

YMMV but wouldn't this be an overkill here?  There must be some (granted, negligible) cost of indirection when using a method.
We want to be able to pass these constants further (in a struct) so the said method would have to mutate the interior state of the struct, therefore we'd also have to initiate it's fields (so probably introduce `new` method), when all we want is a _de-facto_ map with some named fields to silence linter warnings?